client/blocks/login/index.jsx

```diff
@@ -36,7 +36,8 @@ import Notice from 'components/notice';
 import PushNotificationApprovalPoller from './two-factor-authentication/push-notification-approval-poller';
 import userFactory from 'lib/user';
 import SocialConnectPrompt from './social-connect-prompt';
-import JetpackLogo from 'components/jetpack-logo';
+import JetpackHeader from 'components/jetpack-header';
```

Comment:
Nit: would you like to alphabetize the order of these imports?

Reply:
I'm glad to, though they were already not alphabetized, so perhaps it's better to do that in a separate PR?